### ISSUE TITLE
username invalid in prod mode

### ISSUE THREAD

**Issue by aloksinha3110** [akeneo/pim-community-dev#15285]
I am Launching the PIM in prod mode by below cmd
$ `make prod`

the username and password is admin and admin is not working but in dev mode, it is working, may I know what the correct login details for prod mode?

**Comment by Thijzer** [comment]
You can create an admin user easily in the bin/console commands.

`bin/console pim:user:create admin admin support@example.com Admin Admin en_US --admin -n --env=prod`

**Comment by Reiner030** [comment]
yes from documentation since Akeneo 3.2 there is no admin account created anymore.
In 3.1 and 3.2 there was a bug which don't let account created but in 3.2 with a pipe (echo | ... | cat) and sine 4.0 it works correctly.

**Comment by TIM-KALKBRENNER-IT** [comment]
Thanks a lot for your solution, but in my case I had to add a prefix for docker compose:
`docker compose run php bin/console pim:user:create admin admin support@example.com Admin Admin en_US --admin -n --env=prod`
